meetings/2023-08-16.md

```diff
@@ -43,20 +51,117 @@
+* WT: Wanted to join AC on that about representations. What I meant earlier what is not in the current protocol, for GET, ETag is clearly there. Same could go for PUT, based on representation, it can return an ETag.
+* eP: I want to ask about PATCH, since it used a different content type. Which representation would be returned?
+* SC: It depends on that of the representation type of the response; whether the payload has a non-empty content or content-location header, e.g., 201 or 200. It seems like PUT or PATCH responses could thus fall under this case. Is that interpretation correct? What are implementations and browsers currently doing? 
+* SC: POST is, I think, not the case, but it is a bit special in Solid because of container treatment.
+* TBL: The Etag has to do with representation, not with the body of the response but with the state of the resource. With a PUT the state is the body of the request. In a PATCH has changed the state, the client needs to be able to update its Etag. The logic is always: whenever a resource state changes, the Etag changes.
+* AC: Agreed. I just want to make sure what we do does not break how browsers currently behave.
+* TBL: Browsers typically don't use a PUT.
+* AC: I mean, what if we use JavaScript, that's going to use the browsers cache.
+* TBL: It would be useful to have an overview of what we expect browsers to do.
+* SC: Sounds like a moving target. Like CORS. There is so much to do. We have a panel around QA work; we really need people. For us to show interop we need to put the work in, either in the CG or in the WG. That includes the browser behavior: if we have any problems with how the web platform works, we can actually raise those, e.g., to the WICG
+* RG: I want you to read [8.8.3 of RFC9110](https://httpwg.org/specs/rfc9110#field.etag)
+  > The "ETag" field in a response provides the current entity tag for the selected representation, as determined at the conclusion of handling the request. An entity tag is an opaque validator for differentiating between multiple representations of the same resource, regardless of whether those multiple representations are due to resource state changes over time, content negotiation resulting in multiple representations being valid at the same time, or both.
+* SC: I think there is indeed a nuance with the strong and weak Etags. What we currently have in the spec is what we came up with, but it is not set in stone. If you have some time, please read through them to not repeat anything. If there is new material, that might warrant a change.
+* SC: Can anyone interested add to this issue please, and then it might inform more specificity in the Solid protocol, and how much variability we expect here.
+* TBL: The protocol it must give guarantees. The Solid protocol gives guarantees about the state of the resource on the Web. The Etag protocol is the best way of doing so. If it breaks, than you can lose data. We have tests, so that data will still be safe.
```

Comment:
```suggestion
* TBL: The protocol must give guarantees. The Solid protocol gives guarantees about the state of the resource on the Web. The Etag protocol is the best way of doing so. If it breaks, than you can lose data. We have tests, so that data will still be safe.
```